.circleci/config.yml

```diff
@@ -375,7 +375,7 @@ jobs:
       - config-path
       - run:
           name: upload-to-digitalocean
-          command: .travis/upload_to_s3.sh $DO_S3_ACCESS_KEY $DO_S3_SECRET_KEY build/archive/ raiden-ci-test ams3.digitaloceanspaces.com
+          command: .circleci/upload_to_s3.sh $DO_S3_ACCESS_KEY $DO_S3_SECRET_KEY build/archive/ $DO_S3_SUB_SUB_DOMAIN ams3.digitaloceanspaces.com
```

Comment:
Out of curiosity, where are those values defined? `DO_S3_SUB_SUB_DOMAIN` and the others

Reply:
We defined a context with all these environment variables on CircleCI. So in particular the passwords/access tokens do not need to be present in the repo at all, also not in encrypted form as it was with travis.